vllm/entrypoints/openai/api_server.py

```diff
@@ -70,10 +70,10 @@ async def _force_log():
             await asyncio.sleep(10)
             await backend.do_log_stats()
 
-    # if not engine_args.disable_log_stats:
-    #     task = asyncio.create_task(_force_log())
-    #     _running_tasks.add(task)
-    #     task.add_done_callback(_running_tasks.remove)
+    if not engine_args.disable_log_stats:
+        task = asyncio.create_task(_force_log())
+        _running_tasks.add(task)
```

Comment:
if this is here just to be a hard ref to the task so it doesn't get GC'ed, then I think it would work fine as a local variable inside `lifespan` instead of a module-scope variable